README.md

```diff
@@ -1,53 +1,74 @@
-# Run MATLAB® Command
+# Action for Running MATLAB Commands
 
-This action executes a MATLAB script, function, or statement using the _first_
-MATLAB version on the runner's system `PATH`.
+The [Run MATLAB Command](#run-matlab-command) GitHub&reg; action enables you to execute a MATLAB&reg; script, function, or statement on a [GitHub-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) or [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runner:
 
-MATLAB exits with exit code 0 if the specified script, function, or statement
-executes successfully without error. Otherwise, MATLAB terminates with a nonzero
-exit code, which causes the build to fail. You can use the
-[`assert`](https://www.mathworks.com/help/matlab/ref/assert.html) or
-[`error`](https://www.mathworks.com/help/matlab/ref/assert.html) functions in
-the command to ensure that builds fail when necessary. When you use this task,
-all of the required files must be on the MATLAB search path.
+- If you want to use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The action uses the first MATLAB version on the runner's path.
```

Comment:
"The action uses the first MATLAB version on the runner's **system** path."

Reply:
Done.